src/plugins/kibana_overview/public/application.tsx

```diff
@@ -47,6 +47,7 @@ export const renderApp = (
             newsfeed$={newsfeed$}
             solutions={solutions}
             features={features}
+            dataViewEditor={dataViewEditor}
```

Comment:
Why can't the `KibanaOverviewApp` access the `dataViewEditor` dep through context?

Reply:
nit: This is one of those that can become:

```
<KibanaOverviewApp {...{ notifications, http, navigation, newsfeed$ }} ... etc
```